frontend/src/components/Topbar.jsx

```
@@ -103,25 +82,28 @@ export const Topbar = () => {
-
-    // Reset again when component unmounts
     return () => {
       processedIdsRef.current = new Set();
     };
   }, []);
 
   return (
     <div className={styles.container}>
-      <div className={styles.logo} onClick={() => navigate("/dashboard")}>
+      {/* STUDIFY logo */}
+      <button className={styles.logo} onClick={() => navigate("/dashboard")}>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)